test/functional/setup/168256.sql

```diff
@@ -1,6 +1,3 @@
-USE $(dbname) 
-GO
-
```

Comment:
All of these changes are to make the scripts work with DW - but then we are using these scripts for all the other platforms we test on, correct? I assume we've tested these scripts in Bamboo?

Reply:
With this change we no longer need the clumsy workaround outside Windows platforms